API/app/Http/Controllers/Api/ShoppingListController.php

```diff
@@ -3,127 +3,32 @@
 namespace App\Http\Controllers\Api;
 
 use App\Http\Controllers\Controller;
+// Our custom controller for checking permissions (read, update, delete) for lists
```

Comment:
Change to say "Our custom controller for sharing functionality (creating a link or verifying one) for lists".